### PR TITLE
Code style update according to guidelines followed in annapurna project.

### PR DESCRIPTION
**Pull request by cmsivolellaAxesdoors**
Header and source template updated in Header and source files section, function declaration/definition format updated in Formatting section (now the return type lies in the same line as the function declaration/definition).

### CODE REVIEW

File: c_cpp_style.md [hunk 1]
```diff
@@ -312,20 +312,17 @@ literal and explicit cast must be used for constants.
 >     .1 /* no! */
 >     .1f /* no! */
 
-- In the function definition, the return type of a function should be placed in a different line of the function.
+- In the function definition, the return type of a function should be placed in the same line of the function.
 - The function parameters should be on the same function line if they fit; otherwise, wrap arguments at the parenthesis.
 
->     int32_t
->     ComputeActualVelocity(int32_t actualPosition, int32_t lastPosition);
+>     int32_t   ComputeActualVelocity(int32_t actualPosition, int32_t lastPosition);
```
Comment (albertizpe21) [inline c_cpp_style.md]: Could you add that if it's static function this start without caps computeActualVelocity, and if it's public the name of the module in caps underscore, and no caps. PROFILE_computePositionProfile()?

File: c_cpp_style.md [hunk 2]
```diff
@@ -312,20 +312,17 @@ literal and explicit cast must be used for constants.
...
 >      
->     int32_t
->     ComputeRegressionLine(int32_t* pDataArrayX, int32_t* pDataArrayY, 
+>     int32_t   ComputeRegressionLine(int32_t* pDataArrayX, int32_t* pDataArrayY,
```
Comment (albertizpe21) [inline c_cpp_style.md]: and only 1 space between the type and the name of the function?

File: c_cpp_style.md (hunk 1 above)
Comment (cmsivolellaAxesdoors) [reply c_cpp_style.md]: Hecho.

File: c_cpp_style.md (hunk 2 above)
Comment (cmsivolellaAxesdoors) [reply c_cpp_style.md]: Hecho.

**Review by albertizpe21 — APPROVED** [review]

**Pull request merged**